nrfu_tests/test_definition.yaml

```
@@ -0,0 +1,14 @@
+- name: nrfu_tests
+  testcases:
+  - name: test_system_hardware_free_memory
+    description: Testcase for verification of system hardware free memory functionality.

```

```suggestion
    description: Testcase for verification of system hardware free memory utilization.

```
update it at all places